.gitignore

```diff
@@ -31,3 +31,5 @@ build/Release
 config/local.js
 
 .DS_Store
+
+views/layout.ejs
```

Comment:
Ah yes, it changes the JavaScript tags on build. You can tell git not to ignore it, but to assume nothing has changed. We need this in the repository. 

http://gitready.com/intermediate/2009/02/18/temporarily-ignoring-files.html